packages/react/src/returns/hooks/useReturn.ts

```diff
@@ -65,10 +61,9 @@ function useReturn(returnId?: Return['id'], options: UseReturnOptions = {}) {
       : (!!returnEntity || createReturnError) && !isCreatingReturn,
   );
 
-  const fetchReturn = useAction(fetchReturnAction);
-  const resetReturnState = useAction(resetReturnStateAction);
-  const updateReturn = useAction(updateReturnAction);
-  const createReturn = useAction(createReturnAction);
+  const {
+    actions: { fetchReturn, resetReturnState, updateReturn, createReturn },
+  } = useUserReturns({ enableAutoFetch: true });
```

Comment:
`enableAutoFetch` should be false to avoid making a request for user returns unintentionally.